remme/rest_api/certificate.py

```diff
@@ -26,6 +26,9 @@
 from cryptography.hazmat.primitives import hashes
 from cryptography.hazmat.primitives.asymmetric import rsa, padding
 from cryptography.hazmat.backends import default_backend
+from OpenSSL.crypto import PKCS12, X509, PKey
```

Comment:
Not supported yet, but probably going to be supported. Please reference this issue. https://github.com/pyca/cryptography/issues/2860